container/root/tests/nginx/base.goss.yaml

```diff
@@ -1,12 +1,63 @@
 gossfile:
   ../../goss.base.yaml: {}
+
+user:
+  www-data:
+    exists: true
 group:
   www-data:
     exists: true
+
 command:
   nginx -v:
     exit-status: 0
     stderr: []
   nginx -t:
     exit-status: 0
     stderr: []
+
+file:
+  /etc/nginx/sites-available/default:
+    exists: true
+    contains:
+      - listen 8080;
+      - root /var/www/html;
+      - index index.html index.htm;
+      - client_max_body_size 1m;
+      - try_files $uri $uri/ =404;
+      - location ~ /\. {
+      - access_log off;
+      - log_not_found off;
+      - deny all;
+  /etc/nginx/nginx.conf:
+    exists: true
+    contains:
+      - '/;{0}worker_processes\s*(auto|\d+);/'
```

Comment:
@bryanlatten , can you explain one of the example regex's? 
i'm interpreting, say, this line as:
contains a line with "zero semicolons to start", then `worker_processes`, whitespace(s) then `auto` or one or more digits. 

Why is the first `;{0}` required? that part makes no sense to me.